### PR TITLE
[WIP]Bug 1304241, follow-up to PR #1529

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1529

https://bugzilla.redhat.com/show_bug.cgi?id=1304241

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@bfallonf @tnguyen-rh @tpoitras PTAL

**Comment by tnguyen-rh** [comment]
@ahardin-rh LGTM.

File: dev_guide/new_app.adoc [hunk 1]
```diff
@@ -545,13 +545,20 @@ To see all of the configuration options, click the "Show advanced build and depl
 
 ifdef::openshift-online[]
 == Troubleshooting
-`*EmptyDir*` volumes are disabled in OpenShift Online at present. In some situations new-app will automatically create or use templates which contain `*EmptyDir*` volumes. In this scenario you will need to use `oc edit` to modify the relevant PodSpec and replace the use of `*EmptyDir*` volumes with a `*PersistentVolumeClaim*`.
+Currently, `*EmptyDir*` volumes are disabled in OpenShift Online. In some
+situations, `oc new-app` will automatically create or use templates which
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: Do not write "will" unless you plan to indicate "when" something will happen. Suggested edit:

s/will automatically create or use/automatically creates or uses

File: dev_guide/new_app.adoc [hunk 2]
```diff
@@ -545,13 +545,20 @@ To see all of the configuration options, click the "Show advanced build and depl
...
+contain `*EmptyDir*` volumes. In this scenario, you will need to use `oc edit`
```
Comment (tpoitras) [inline dev_guide/new_app.adoc]: s/will need to/must

**Comment by tpoitras** [comment]
Couple small nits about future tense. Other than that, LGTM! :ghost:

**Comment by ahardin-rh** [comment]
QA failed. Followed up with @dgoodwin in the BZ.

**Comment by ahardin-rh** [comment]
The work here is no longer valid. Closing in favor of the work done in the following PRs:
https://github.com/openshift/openshift-docs/pull/1760
https://github.com/openshift/openshift-docs/pull/1759
https://github.com/openshift/openshift-docs/pull/1758

**Pull request closed without merging**